test/language/class/class-basics-extends.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class basics extends
+---*/
+class C extends null {}
```

Comment:
This one is not valid with the new syntax. If there is an extends expression then the default constructor is:

```js
constructor(...args) {
  super(..args);
}
```

which throws because `Function.prototype` is not constructable.